src/API.js

```diff
@@ -24,6 +24,10 @@ class API {
       });
     });
   };
+
+  static setContactData = (data) => {
+    db.collection("Contact-us").add(data).then(console.log("data added"));
```

Comment:
change the collection name to lower case snake case
```suggestion
    db.collection("contact_us").add(data).then(console.log("data added"));
```

Reply:
remove the console and change it that the modal say `sent` or `thanks for contacting us` and then disappearing

Reply:
I will fix the .then and send the email using EmailJS but is it okay if I keep it Contact_us for now, because i couldn't rename and I would have to rewrite the whole db because they are all capital letter